extension/assets/.devcontainer/axon-ivy/.env

```diff
@@ -0,0 +1 @@
+export IVY_HOME="/usr/lib/axonivy-engine"
```

Comment:
no idea if that works on windows, have to check

Reply:
Should work, because dev images are working with wsl I think: https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers